yb-voyager/src/callhome/diagnostics.go

```diff
@@ -0,0 +1,159 @@
+package callhome
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+//call-home json formats
+var (
+	jsonFilePath    string
+	Payload         payload
+	jsonBuf         []byte
+	SendDiagnostics bool
+)
+
+const(
+	CALL_HOME_SERVICE_HOST      = "34.83.149.226"
```

Comment:
I assume this is a static IP address.
Note that ephemeral external IP addresses of VMs change on reboot.

Reply:
We have reserved a static IP address for the VM
All details can be found here: https://console.cloud.google.com/compute/instancesDetail/zones/us-west1-b/instances/yb-voyager-callhome?project=yugabyte&pageState=(%22duration%22:(%22groupValue%22:%22PT1H%22,%22customValue%22:null))